templates/zerver/help/invite-new-users.md

```diff
@@ -8,16 +19,14 @@ When you invite users, you can:
 * Configure which streams they will be added to. The organization's
   [default streams](/help/set-default-streams-for-new-users) will be preselected.
 
-Your organization may also want to configure [default
-settings](/help/configure-default-new-user-settings) for new users.
+Organization administrators can
+[configure](/help/restrict-account-creation#change-who-can-send-invitations)
+which [roles](/help/roles-and-permissions) have permission to invite users to
+the organization. You will only see an **Invite users** menu option if you have
+permission to invite users.
 
 ## Send e-mail invitations
```

Comment:
I just noticed that we use "e-mail" in this subheader. Is that intentional or should we update it to be "email" which we use throughout the rest of the page?